### ISSUE TITLE
Rebel Strike - Corrupt HUD

### ISSUE THREAD

**Issue by GB-CG** [FIX94/Nintendont#352]
It happens when you play on a level, first try, than aborting the mission and retry, it somehow corrects itself and not sometimes.

![- screen shot 21 12 2016 02 19](https://cloud.githubusercontent.com/assets/11931125/21393324/d7823c92-c78b-11e6-9c80-07852cf8f313.png)

Tested on v4.432. Verified MD5.

**Comment by SieKensou** [comment]
Appart of that problem with the HUD I noticed that when the sequences before/between/after missions are playing it has some pauses when the screen turns black to change the camera plane. These times are more reduced when playing with the original game.

**Comment by SlyCooperReloadCoded** [comment]
Downloaded the latest build, this is still an issue.  DIOS MIOS doesn't have this bug, but I'd rather use this.

**Comment by mikesk8** [comment]
Hello, 

I have just installed the latest Nintendont version and this bug still exists :( I have PAL WiiU and the installed game is NTSC/US one. 
![IMG_8047](https://user-images.githubusercontent.com/46671688/206928856-706375a2-08a4-43c4-b29c-5dc21ed75796.JPG)

Is there any known solution? or can I do any additional tests that could help to solve it?

Thank you, M.

**Comment by carnage702** [comment]
> Hello,
> 
> I have just installed the latest Nintendont version and this bug still exists :( I have PAL WiiU and the installed game is NTSC/US one. ![IMG_8047](https://user-images.githubusercontent.com/46671688/206928856-706375a2-08a4-43c4-b29c-5dc21ed75796.JPG)
> 
> Is there any known solution? or can I do any additional tests that could help to solve it?
> 
> Thank you, M.

there is nothing to be done, this was never solved, its a random gpu bug that was left and no one could fix it, it doesnt make the game less playable and you can reset the bug but restarting missions and so on.

**Comment by mikesk8** [comment]
Thank you for fast answer. Do you know maybe if this occurs also on Wii (and not on WiiU) playing via Nintendont? I don't recall I have seen it on Wii playing from the original disc though.

**Comment by mikesk8** [comment]
Thank you for the answer. Let us wait for somebody who can fix this in the future :)

**Comment by hurda** [comment]
Just wanted to leave these links here from swiss which mention some decompressing of the game's DOL, which might help with this issue whenever somebody wants to try fixing it.

[Error loading save for Star Wars Rogue Squadron III - Rebel Strike from physical memory card](https://github.com/emukidid/swiss-gc/issues/465)

>No. The problem is a hack we use to get around implementing the memory card's security mechanism isn't being applied.
>Note that neither Dolphin nor Nintendont implement this mechanism.

[Patch Star Wars Rogue Squadron III: Rebel Strike's compressed VM data.](https://github.com/emukidid/swiss-gc/commit/fe356acf8539e5f122d4fbebe4824492ad15c2e2)

Might even explain [Error loading save for Star Wars Rogue Squadron III - Rebel Strike from physical memory card](https://github.com/FIX94/Nintendont/issues/946)

**Comment by mikesk8** [comment]
It would be really amazing if somebody could fix it. Personally, it is my top GC game, and this error is quite annoying ;-(

**Comment by carnage702** [comment]
if you find someone that can fix it and make a pull request i will merge it gladly, i dont have the knowledge to fix this sadly.

**Comment by mikesk8** [comment]
Unfortunately, I do not know anybody who can help; and I don't have programming skills :( 

Maybe some skilled Star Wars fan helps us in the future ... ;-)

**Comment by SlyCooperReloadCoded** [comment]
These games are so cursed, we're decades away from being able to emulate them and real hardware game loaders can't handle them properly.  You pretty much have to run the games from disc and real memory card which is just sad.

**Comment by mikesk8** [comment]
Yes, I have seen it working ok on Swiss; however, not an option for WiiU :( Maybe somebody just fixes it here too ;-)

**Comment by TwilightSlick** [comment]
This graphic bug does not occur in Devolution, however the load times are much longer there for some reason.

**Comment by mikesk8** [comment]
We just need somebody with programming skills to fix it here ;-)

**Comment by mariomadproductions** [comment]
@GB-CG issues shouldn't really be closed unless they've been fixed. otherwise what's the point of an issue tracker?
